docs/guide/interactions/slash_commands.rst

```diff
@@ -0,0 +1,1232 @@
+
+Since guild commands can be useful in a development scenario, as often we don't want unfinished commands
+to propagate to all guilds, the library offers a helper method :meth:`.CommandTree.copy_global_to`
+to copy all global commands to a certain guild for syncing:
+
+.. code-block:: python
+
+    class MyClient(discord.Client):
+        def __init__(self):
+            super().__init__(intents=discord.Intents.default())
+            self.tree = app_commands.CommandTree(self)
+
+        async def setup_hook(self):
+            guild = discord.Object(695868929154744360) # a testing server
+            self.tree.copy_global_to(guild)
+            await self.tree.sync(guild=guild)
+
+You'll typically find this syncing paradigm in some of the examples in the repository.
+
+Checks
+-------
+
+Checks refer to the restrictions an app command can have for invocation.
+A user needs to pass all checks on a command in order to be able to invoke and see the command on their client.
+
+Age-restriction
+++++++++++++++++
+
+Indicates whether this command can only be used in NSFW channels or not.
+
+This can be configured by passing the ``nsfw`` keyword argument within the command decorator:
+
+.. code-block:: python
+
+    @client.tree.command(nsfw=True)
+    async def evil(interaction: discord.Interaction):
+        await interaction.response.send_message("******") # very explicit text!
+
+Guild-only
++++++++++++
+
+Indicates whether this command can only be used in guilds or not.
+
+Enabled by adding the :func:`.app_commands.guild_only` decorator when defining an app command:
+
+.. code-block:: python
+
+    @client.tree.command()
+    @app_commands.guild_only()
+    async def serverinfo(interaction: discord.Interaction):
+        assert interaction.guild is not None
+        await interaction.response.send_message(interaction.guild.name)
+
+Default permissions
+++++++++++++++++++++
+
+This sets the default permissions a user needs in order to be able to see and invoke an app command.
+
+Configured by adding the :func:`.app_commands.default_permissions` decorator when defining an app command:
+
+.. code-block:: python
+
+    @client.tree.command()
+    @app_commands.default_permissions(manage_nicknames=True)
+    async def nickname(interaction: discord.Interaction, newname: str):
+        guild = interaction.guild
+        if not guild:
+            await interaction.response.send_message("i can't change my name here")
+        else:
+            await guild.me.edit(nick=newname)
+            await interaction.response.send_message(f"hello i am {newname} now")
+
+Commands with this check are still visible and invocable in the bot's direct messages,
+regardless of the permissions specified.
+
+To prevent this, :func:`~.app_commands.guild_only` can also be added.
+
+.. warning::
+
+    This can be overriden to a different set of permissions by server administrators through the "Integrations" tab on the official client,
```

Comment:
```suggestion
    This can be overridden to a different set of permissions by server administrators through the "Integrations" tab on the official client,
```